### ISSUE TITLE
Fix false positives for interpolation in function-calc-no-invalid

### ISSUE THREAD

**Issue by evilebottnawi** [stylelint/stylelint#3908]
> Clearly describe the bug

False positive on `scss` interpolation

> Which rule, if any, is the bug related to?

e.g. `function-calc-no-invalid`

> What CSS is needed to reproduce the bug?

e.g.

```scss
a {
  width: calc(
    #{$min-value} +
    #{strip-unit($max-value - $min-value)} *
    (100vw - #{$min-vw})
  );
}
```

> What stylelint configuration is needed to reproduce the bug?

e.g.

```json
{
  "rules": {
    "function-calc-no-invalid": true
  }
}
```

> Which version of stylelint are you using?

`9.10.0`

> How are you running stylelint: CLI, PostCSS plugin, Node API?

CLI

> Does the bug relate to non-standard syntax (e.g. SCSS, Less etc.)?

SCSS

> What did you expect to happen?

No warnings to be flagged.

> What actually happened (e.g. what warnings or errors did you get)?

```shell
src/shared/styles/mixins/_fluid-type.scss
 4:42  ✖  Expected a valid expression   function-calc-no-invalid
```

**Comment by evilebottnawi** [comment]
Also
```scss
a {
  width: calc(
    #{$min-value} +
    #{$max-value} *
    (100vw - #{$min-vw})
  );
}
```

Output:

```shell
src/shared/styles/mixins/_fluid-type.scss
 3:11  ✖  Expected space before "-" operator   function-calc-no-invalid
 3:12  ✖  Expected space after "-" operator    function-calc-no-invalid
 4:11  ✖  Expected space before "-" operator   function-calc-no-invalid
 4:12  ✖  Expected space after "-" operator    function-calc-no-invalid
 5:20  ✖  Expected space before "-" operator   function-calc-no-invalid
 5:21  ✖  Expected space after "-" operator    function-calc-no-invalid
```

Maybe related.

Looks we don't avoid interpolation.
